test/restart.jl

```diff
@@ -74,6 +64,11 @@ function _error(
     return error
 end
 
+function rmap(val; ignore = Set([:rc]))
+    properties = filter(x -> !(x in ignore), propertynames(val))
+    properties = propertynames(val)
+end
```

Comment:
@Sbozzolo, I think that this might be leading to the Revise issue?

Reply:
Weird. I removed it, we are not even using it anymore.